.eslintignore

```diff
@@ -2,7 +2,9 @@ build
 coverage
 demos/aws-sdk
 demos/electron/*
-demos/browser/*
+demos/browser/*.*
+demos/browser/app/meetingV2/*.*
```

Comment:
it was not linting before.

Because we exclude the path first and then use `!` to include. but https://git-scm.com/docs/gitignore#_pattern_format does not work that way (parent directory is already excluded).

This file change is just to include "videofilter" directory, I plan to fix the pattern gradually.